modules/deploying-a-model-using-the-multi-model-serving-platform.adoc

```diff
@@ -18,7 +18,8 @@ ifdef::upstream[]
 endif::[]
 * You have enabled the multi-model serving platform.
 * You have created a data science project and added a model server.
-* You know the folder path for the data connection that you want the model to access.
+* You have access to S3-compatible object storage.
+* You know the folder path for the datasets in your S3-compatible object storage bucket, that you want your model to access.
```

Comment:
suggest: For the datasets that you want your model to access, your know the folder paths in your S3-compatible object storage bucket.

Reply:
Addressed in the next commit.